src/pages/AppDetail.svelte

```diff
@@ -33,6 +34,13 @@
   }
 </style>
 
+{#if app}
+  {#if app.prototype}
```

Comment:
I believe this banner can be rendered after the metadata table. Can you please also add just a bit more top and bottom margin?